corehq/apps/users/account_confirmation.py

```diff
@@ -85,7 +85,11 @@ def send_account_confirmation_sms(commcare_user):
 
 def _get_account_confirmation_template_params(commcare_user, message_token, url_name):
     url = absolute_reverse(url_name, args=[commcare_user.domain, message_token])
+    name = ""
+    if commcare_user.first_name or commcare_user.last_name:
+        name = " ".join([commcare_user.first_name, commcare_user.last_name])
     return {
+        'name': name,
```

Comment:
There's actually a helper for this already
```suggestion
    return {
        'name': commcare_user.full_name,
```
Though you may want to use `commcare_user.human_friendly_name` so it's not blank if first and last name aren't specified.  It falls back to username in that case.

Reply:
Username is already included in the message body and would be repetitive in case of fallback. As per received requirements from project, using `full_name` would be apt. Thanks.